### ISSUE TITLE
I would like to add this feature Give your DSA journey a proper guide using pathways in the website.

### ISSUE THREAD

**Issue by aashi2912** [Algo-Phantoms/Algo-Phantoms-Frontend#147]
We can add one proper section in home page about this feature and get started to build this one.
I would like to add this section in home page as **RoadMap to your DSA journey.**

and we can further add proper and more details in this section.

GSSOC'21 . If you like it or want to discuss about this topic please assign it to me. @todi-2000 @diyajaiswal11 @Shruti3004 @rudrakshi99

**Comment by ravikr126** [comment]
yes you can work on it .use proper roadmaps for a person who starts their journey from beginner to expert level
@aashi2912

**Comment by aashi2912** [comment]
hey, what should I do properly in this? Can u clarify as you put label as documentation?

**Comment by abstrxtInfinity** [comment]
The Roadmap to DSA is not styled enough, I would like to work on this. 
GSSOC'21 participant  . If you like please assign it to me. @todi-2000 @diyajaiswal11 @Shruti3004 @rudrakshi99

![image](https://user-images.githubusercontent.com/55052738/111413422-401a0c80-8704-11eb-81f7-ec3bbc2a2c94.png)

**Comment by aashi2912** [comment]
This works fine on my browser. 
![DSA1](https://user-images.githubusercontent.com/52272710/111414730-79537c00-8706-11eb-955f-b284eebd29af.png)

Maybe check your browser for latest version?

**Comment by abstrxtInfinity** [comment]
My browser has latest version.

**Comment by aashi2912** [comment]
Okay, Maybe there will be some problem. Fix it.

**Comment by abstrxtInfinity** [comment]
I would fix it. Please assign me the issue #183

**Comment by abstrxtInfinity** [comment]
I have fixed this issue. This is how it looks now, hoping its up to the mark. Do share if there is anything else that is to be added or changed.
![image](https://user-images.githubusercontent.com/55052738/111431824-4323f580-8722-11eb-947c-139e4c56ef69.png)

![image](https://user-images.githubusercontent.com/55052738/111431847-48814000-8722-11eb-9cfb-9398b211a3fd.png)

**Comment by aashi2912** [comment]
GREAT !!

**Comment by SilverGraph** [comment]
issue: #183 

I would like to work on styling the website as well and this is what I came up with:

![Screenshot 2021-03-18 181019](https://user-images.githubusercontent.com/74287708/111627589-37fdc200-8815-11eb-9b46-e2c88e2e625f.png)

Please allow me to contribute to this project  @todi-2000 @diyajaiswal11 @Shruti3004 @rudrakshi99

**Comment by ravikr126** [comment]
> I have fixed this issue. This is how it looks now, hoping its up to the mark. Do share if there is anything else that is to be added or changed.
> ![image](https://user-images.githubusercontent.com/55052738/111431824-4323f580-8722-11eb-947c-139e4c56ef69.png)
> 
> ![image](https://user-images.githubusercontent.com/55052738/111431847-48814000-8722-11eb-9cfb-9398b211a3fd.png)

you can work now @abstrxtInfinity

**Comment by abstrxtInfinity** [comment]
> you can work now @abstrxtInfinity

I have completed the task and have staged the changes for review, PR - #185
